src/model/Game.java

```diff
@@ -0,0 +1,111 @@
+package model;
+
+import utils.Config;
+
+
+// include move, alert, rest, search, hide, resetDay, resetWeek, startDay, gameOver
+public class Game {
+
+    Player[] players = new Player[Config.MAX_CLIENTS] ;
+    int playerCount  = 0;
+    int currentDay   = 1;
+
+    // only adds a player if there's enough room for a new one
+    // does nothing if the game is already full
+    public boolean addPlayer(Player player) {
+        boolean canAdd = false;
+        if (playerCount < players.length) {
+            players[playerCount] = player;
+            playerCount++;
+            canAdd = true;
+        }
+        return canAdd;
+    }
+
+    public int getPlayerCount() {
+        return playerCount;
+    }
+
+    // returns true if the player is allowed to move to the clearing
+    public boolean move(Player player, Clearing newClearing) {
+        player.setHidden(false);
+        boolean canChange =  (player.getLocation().canChangeClearing(newClearing));
+        if (canChange)
+            player.setLocation(newClearing);
+        return canChange;
+    }
+
+    public boolean hide(Player player) {
+        // todo: check if they can hide
+        player.setHidden(true);
+        return true;
+    }
+
+    // player can can alert or unalert a weapon
+    public void alert(Weapon weapon, boolean alert) {
+        weapon.setActive(alert);
+    }
+
+    public void rest(Player player) {
+        player.setFatigue(0); // I'm assuming it resets the fatigue, which I'm pretty sure is wrong
+    }
+
+    public void resetDay() {
+        if (currentDay == 28) {
+            endGame();
```

Comment:
if it is day 28, we might just want to send back false instead of going right into `endGame`. That way the control would know to do `endGame` instead which returns the winning player
what do you think @michaelcatt ?